Create PM article

rohit-ganguly opened the pull request:
closes #207 #397 

Finally lol, an article on Product Management. I wanted to take a fundamentals approach as opposed to focusing on what the role actually does (enough of the '6 figures without coding' yapping online!!)

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+## The Important Basics
+
+Product Management looks different *everywhere*. Sticking to the fundamentals will take you very far in your PM career no matter what exactly Product looks like in your organization. 
```
"no matter what exactly Product looks like in your organization" -> "no matter what the role looks like in your organization"

Ease of reading. Would this edit make sense?

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+The first thing to remember is that the term Product Management can be defined by the three words "who", "what", and "why". Everything else in the career is to gain a clearer understanding of these three concepts.
+
+To elaborate:
+- Who (exactly) are we building this for?
```
I'd suggest bolding "Who", "What", "Why". 

Idk what you mean be "(exactly)", you also used that word in the previous section, so is it important?

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+- What (exactly) are we doing? What does success look like with this feature?
+- Why should we do this (and not something else)?
+
+In the scope of an engineering team, the job of the Product Manager is to clarify the work to be done, and have a clear reason why. You aren't the one defining how the problem gets solved at the end of the day (design and engineering-wise). You can come to the table with ideas, but designers and engineers are there to handle that portion - you need to ensure the end result is a good answer to 'wh ...
```
"result" -> "product" or "solution"?

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+Product Management looks different *everywhere*. Sticking to the fundamentals will take you very far in your PM career no matter what exactly Product looks like in your organization. 
+
+### The first fundamental
```
Doesn't follow capitalization scheme

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+In the scope of an engineering team, the job of the Product Manager is to clarify the work to be done, and have a clear reason why. You aren't the one defining how the problem gets solved at the end of the day (design and engineering-wise). You can come to the table with ideas, but designers and engineers are there to handle that portion - you need to ensure the end result is a good answer to 'wh ...
+
+### The second fundamental
```
Doesn't follow capitalization scheme

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+You're a Product Manager on a software product. Let's say you're focused on Business to Consumer (B2C) products like Netflix.
+
+My personal approach is to start backwards and always ask questions. What makes your company money? For Netflix, when focusing on customer features, it's  membership fees. Why do customers pay money for Netflix? Because they find content they want to watch. How do they find content to watch? Through the algorithm that suggests categories and shows. After that, you can dig deeper into specifics. W ...
```
Extraneous space between "it's" and "membership"

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+My personal approach is to start backwards and always ask questions. What makes your company money? For Netflix, when focusing on customer features, it's  membership fees. Why do customers pay money for Netflix? Because they find content they want to watch. How do they find content to watch? Through the algorithm that suggests categories and shows. After that, you can dig deeper into specifics. W ...
+
+Understanding how your business makes money, now what are the higher ups saying strategy-wise? What can your product do to align with  overall business goals? While you represent the customers, you also have to get buy-in and aligning with strategy is a must.
```
Extraneous space between "with" and "overall"

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+### Ideation
+
+You'll eventually bring your prioritized ideas to your stakeholders (design, engineering, leadership, etc) and they'll have their own thoughts on the question they're responsible for, *how* do we get it done? 
```
"thoughts on the question" seems kinda out of place

NathanBarta replied:
What's "it"?

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+You can come to the table with ideas on how it should be done from a customer's perspective. Understand their job to be done (JTBD) and pain points. The design and engineering teams will handle the rest - do *not* get attached to a solution! 
+
+The most important thing is adjusting your ideas with impact/effort. Again, you are not responsible for ideating a solution - your job is to ensure that there is value provided to customers at the end of the day. If you're getting pushback on your features and you can't defend it, you need to develop a deeper understanding of the customers/problem/business/competition OR improve your communicatio ...
```
"ideas with impact/effort"

Is "with" the right word?

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+The most important thing is adjusting your ideas with impact/effort. Again, you are not responsible for ideating a solution - your job is to ensure that there is value provided to customers at the end of the day. If you're getting pushback on your features and you can't defend it, you need to develop a deeper understanding of the customers/problem/business/competition OR improve your communicatio ...
+
+At the end of it all, you'll have to prioritize scoped features (features that have had their effort to build measured). Ideally you go for low-effort high-impact features first, and work your way backwards to medium-effort high-impact, medium-effort medium-impact, etc. This is very situational but this process will look the same everywhere.
```
Maybe bring up the concept of "scope" in the first paragraph of this section when you are talking about the other stakeholders.

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,91 @@
+
+You should have a clear reason for prioritizing work, and if you're feeling unsure in a particular scenario, you have more to learn about your customer, the problem, your competitors, or the business.
+
+### Ideation
```
Title scheme not followed. Should be "Problem Ideation".

NathanBarta commented:
Looks good! Definitely fills a hole in the wiki content, so thank you for the contribution! 

I've left a few minor comments, feel free to ping me if you need clarification on any of them or if you'd like to discuss.

rohit-ganguly commented:
> Looks good! Definitely fills a hole in the wiki content, so thank you for the contribution!
> 
> I've left a few minor comments, feel free to ping me if you need clarification on any of them or if you'd like to discuss.

Addressed your feedback - let me know your thoughts! Thanks :)

NathanBarta commented on src/guides/career/product-management.md:
```diff
@@ -0,0 +1,96 @@
+
+### Problem Prioritization
+
+Ideally, this is a pretty straightforward answer once you have a good understanding of the problems to solve. This generally can be prioritized by looking at why you're building it. If you're building something to satisfy users, build something that satisfies a more important pain point. If you're looking to make money, pick the feature that has a higher chance to increase money made from your pr ...
```
Paragraph confuses me. Did you mean "answered" instead of "prioritized" in sentence 2?
